cmd/livepeer_bench/livepeer_bench.go

```diff
@@ -151,8 +163,22 @@ func main() {
 		time.Sleep(300 * time.Millisecond)
 	}
 	wg.Wait()
-	fmt.Fprintf(os.Stderr, "Took %0.4v seconds to transcode %v segments of total duration %vs (%v concurrent sessions)\n",
-		time.Since(start).Seconds(), segCount, segTotalDur, *concurrentSessions)
+	statsTable := tablewriter.NewWriter(os.Stderr)
+	stats := [][]string{
+		{"Concurrent Sessions", fmt.Sprintf("%v", *concurrentSessions)},
+		{"Total Segs Transcoded", fmt.Sprintf("%v", segCount)},
+		{"Real-Time Segs Transcoded", fmt.Sprintf("%v", realTimeSegCount)},
+		{"* Real-Time Segs Ratio *", fmt.Sprintf("%0.4v", float64(realTimeSegCount)/float64(segCount))},
```

Comment:
Minor: segCount and srcDur could be 0 if the user passes in a playlist with no segments. This would be a case of user error. Good practice to add a check for that to avoid a divide by zero panic though.